src/ed_object_models/spawn_sdf_in_gazebo.py

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/python
+
+import rospy
+import yaml
+import os
+import glob
+
+from rospkg import RosPack
+from gazebo_msgs.srv import SpawnModel
+from geometry_msgs.msg import Pose
+
+
+def from_yaml(yaml_file):
+    ''' Spawns a list of sdf files from a yaml file into Gazebo.
+    :param yaml_file:
```

Comment:
It's likely helpful if you specify expected yaml data format and meaning of fields. This is likely lacking with many of the files though.

Reply:
I have updated my docstring. Do you think that this is clear enough? This should also make the sdf branch on fast_simulator_data obsolete.